pkg/controller/config/operator_config.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import (
+	"flag"
+)
+
+var OperatorConfig Config
+
+type Config struct {
+	AtlasDomain          string
+	EnableLeaderElection bool
+	MetricsAddr          string
+}
+
+// ParseConfiguration fills the 'OperatorConfig' from the flags passed to the program
+func ParseConfiguration() {
+	OperatorConfig = Config{}
+	flag.StringVar(&OperatorConfig.AtlasDomain, "atlas-domain", "https://cloud-qa.mongodb.com", "the Atlas URL domain name (no slash in the end).")
```

Comment:
idk maybe cloud.mongodb.com should be the default?

Reply:
yes, was a bit lazy but I think you are right